docs/quick_start.rst

```diff
@@ -61,3 +62,9 @@ Execute the following Python code.
     pymeasure.__version__
 
 You should see the version of PyMeasure printed out. At this point you have PyMeasure installed, and you are ready to start using it! Are you ready to :doc:`connect to an instrument <./tutorial/connecting>`?
+
+.. warning::
+    The release of numpy version 2 caused some compatibility issues.
+    PyMeasure is compatible with version 2 as of PyMeasure version 0.14.
+    To use PyMeasure with numpy>=2, you also ensure the dependencies are compatible.
```

Comment:
Why does the user _have to_ do this? Should not our package metadata make sure of this automatically?

Reply:
I'm not sure if we should even mention this whole warning as long as our package specification is correct? 
This whole paragraph will become outdated really quickly, so we need to remember to remove it again. Finding pandas and pint versions compatible with both pymeasure _and_ numpy 2 is the responsibility of the package manager (pip/conda). If that fails, it's ultimately the issue of a dependency, not ours, right?

Reply:
You're right, we should not add this warning.

If numpy 2.0 is already installed, it will fail until the dependencies are met. Otherwise, it won't install numpy 2.0, so it won't be an issue either.

Reply:
Sure, I'll remove the warning.

Note that we need to update the requirements.yml file as well once the final version of both numpy=2.0.0 and pint>0.23 are released.